config

```diff
@@ -76,6 +76,9 @@
 # Print extra information to help debugging problems (boolean value)
 #debug=<true/false>
 
+# Don't report progress (boolean value)
+#no-progress=<true/false>
```

Comment:
you should use underscore here: no_progress, it will be converted to no-progress at parsing time.